bin/prompt-command.js

```diff
@@ -60,15 +59,6 @@ module.exports = function promptForInstallation(packages, ...args) {
 			options[0] = "add";
 		}
 
-		if (packages === "init") {
```

Comment:
why is this line removed?

Reply:
Doing local installation for `init` as decided, so this `if` is not required now in which we are replacing `-D` with global equivalents.

Reply:
I think we are allowing global installation as well

Reply:
Refer
https://github.com/webpack/webpack-cli/pull/742#issuecomment-460654658

Here you said for local setup.

Currently we are not allowing global installation in this scenario.

Reply:
Oh sorry. So we need global installation. This PR needs to support that

Reply:
Okay :+1:  will do it.